Sources/Jaeger/BaggageContext+TraceContext.swift

```diff
@@ -0,0 +1,30 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Jaeger Client Swift open source project
+//
+// Copyright (c) 2020 Moritz Lang and the Jaeger Client Swift project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import Baggage
+import W3CTraceContext
+
+extension Baggage {
+    private enum TraceContextKey: Baggage.Key {
+        typealias Value = TraceContext
+    }
+
+    public var traceContext: TraceContext? {
```

Comment:
that sound sounds okey -- so the W3CTraceContext does not deal with baggage that's okey.

Please add some docs here?